configs/src/deployments/osx-ens-domains.ts

```diff
@@ -0,0 +1,14 @@
+import {SupportedNetworks} from '../networks';
+import {ENSNetworkDomain, ENSNetworkDomainsMap} from './types';
+
+export const commonDomain: ENSNetworkDomain = {
+  daoEns: 'dao.eth',
+  pluginEns: 'plugin.dao.eth',
+};
+
+export const exceptionalDomains: ENSNetworkDomainsMap = {
```

Comment:
Since this is part of all networks why not adding it as a prop of the Networks in `NetworkConfigs`

Reply:
Adding something like this to the networks object @Rekard0 @jordaniza lmk what you think


```ts
export const networks: NetworkConfigs = {
  [SupportedNetworks.MAINNET]: {
    url: 'https://rpc.tenderly.co/fork/d3168a50-0941-42e2-8b9b-bf544c60c356',
    isTestnet: false,
    chainId: 1,
    name: SupportedNetworks.MAINNET,
    aliases: {
      ethers5: 'homestead',
    },
    ens: {
        dao: 'dao.eth',
        plugin: 'plugin.dao.eth'
    }
  }
  // ... other nw
}

Reply:
> Since this is part of all networks why not adding it as a prop of the Networks in NetworkConfigs

@Rekard0 and I discussed it but we thought that the ENS domain was not directly related to the network configuration but to the deployment.

Reply:
I mean, it could be seen as network related, and indeed it is, but is not a network configuration itself.

Instead, it is needed when deploying to link it to the new contract, so we saw it more like deployment stuff.

Said that, I have no strong preference for either option, I think they both could be reasonable.